test/unit/fixture/reverse_with_layer_filtering_non_coarse_subset.js

```diff
@@ -0,0 +1,41 @@
+var vs = require('../../../query/reverse_defaults');
+
+module.exports = {
+  'query': {
+    'bool': {
+      'filter': [
+        {
+          'geo_distance': {
+            'distance': '500km',
```

Comment:
Yeah I like that plan (I mentioned it in my overall comment on the review). Basically I'm asking two things:

1.) Can you confirm that the 500km here comes from line 141 in `test/unit/query/reverse.js` and
2.) Can we remove that line or lower the value?